mubench.pipeline/tests/tasks/implementations/test_check.py

```diff
@@ -0,0 +1,18 @@
+from nose.tools import assert_equals
+from unittest.mock import MagicMock
+
+from tasks.implementations.check import Check
+from tests.test_utils.data_util import create_project
+
+
+class TestCheck():
+    def test_process_project(self):
+        assert_equals([], Check().process_project(create_project("-project-")))
+
+    def test_checks_requirements_on_start(self):
+        uut = Check()
+        uut._check_all_requirements = MagicMock()
+
+        uut.start()
+
+        uut._check_all_requirements.assert_called_once_with()
```

Comment:
This test basically just tests that the method you invoke is invoked. I don't see value in this. Can you mock the `Requirements.__subclasses__()` instead and return some test stub to control what happens?